ipa_map_comparison/include/ipa_map_comparison/ipa_map_comparison_node.h

```diff
@@ -0,0 +1,32 @@
+#ifndef MAP_COMPARISON_NODE_H
```

Comment:
Add license headers to all files. You can just copy/paste it from other nav pkgs